bitcoin/src/bip32.rs

```diff
@@ -683,7 +683,7 @@ impl ExtendedPubKey {
 
     /// Attempts to derive an extended public key from a path.
     ///
-    /// The `path` argument can be both of type `DerivationPath` or `Vec<ChildNumber>`.
+    /// The `path` argument can be both of type `DerivationPath` or `AsRef<ChildNumber>`.
```

Comment:
"can be of any type implementing `AsRef<ChildNumber`" would be a bit more natural.

Reply:
Fixed in https://github.com/rust-bitcoin/rust-bitcoin/pull/1658/commits/f69363d71ad64046c7ebf94a603d6e923f66dd51